.github/scripts/release.py

```diff
@@ -224,7 +224,7 @@ def restorePriorToRelease(self):
 
 
 class KernelRelease(BaseRelease):
-    def __init__(self, mGit, version, commit='HEAD', git_ssh=False, git_org='FreeRTOS', repo_path=None, branch='main', main_br_version='', do_not_push=False):
+    def __init__(self, mGit, version, commit='HEAD', git_ssh=False, git_org='FreeRTOS', repo_path=None, branch='main', main_br_version='', do_not_push=False, do_not_update_src=False):
```

Comment:
Nit: Flip this booleans value and naming. You're later doing `if do_not_update_src == False`. Why do these negation checks? If you simply named if `update_src` your check would be as simple as `if update_src`. This would also increase readability.